pkg/data/domain/cluster/service.go

```diff
@@ -0,0 +1,29 @@
+package cluster
+
+import (
+	"github.com/giantswarm/microerror"
+
+	"github.com/giantswarm/kubectl-gs/pkg/data/client"
+)
+
+var _ Interface = &Service{}
```

Comment:
This construct helps us check during compilation if the `Service` struct satisfies the `Interface` interfac